foundry/hooks.d.ts

```diff
@@ -740,12 +747,30 @@ declare namespace Hooks {
     userId: number
   ) => void;
 
-  /**
-   * This is called when the official world time changes.
-   * @param worldTime - the new world time
-   * @param dt        - the time advance delta, in seconds
-   * @remarks The name is 'updateWorldTime'.
-   * @see {@link GameTime#onUpdateWorldTime}
-   */
-  type UpdateWorldTime = (worldTime: number, dt: number) => void;
+  type DynamicCallbacks =
+    | CloseApplication
+    | ControlPlaceableObject
+    | CreateEmbeddedEntity
+    | CreateEntity
+    | DeleteEmbeddedEntity
+    | DeleteEntity
+    | GetApplicationHeaderButtons
+    | GetChatLogEntryContext
+    | GetCombatTrackerEntryContext
+    | GetCompendiumDirectoryEntryContext
+    | GetPlaylistDirectorySoundContext
```

Comment:
Is this called differently (i.e. no "Entry") intentionally? The remark speaks about `EntryContext`, is that wrong?

Reply:
No that's me writing the docs wrong.